src/solver/src/mesh_builder.C

```diff
@@ -98,20 +81,18 @@ namespace GRINS
       }
 
     // Create UnstructuredMesh object (defaults to dimension 1).
-    libMesh::UnstructuredMesh* mesh;
+    std::shared_ptr<libMesh::UnstructuredMesh> mesh;
```

Comment:
Why shared_ptr and not unique_ptr?  We'd expect to have one clear "owner" of the mesh throughout its lifetime, no?

Reply:
Yeah, it's an atavism. It should be a unique_ptr. I think when this code was written originally, there was no UniquePtr wrapper in libMesh, but there was a SharedPtr wrapper that later got converted to std::shared_ptr. I think. It was a while ago.

Reply:
I didn't change it now because that will touch other pieces of code. Was trying to keep the PR self-contained.